Backend/Backend/Program.cs

```diff
@@ -94,9 +99,6 @@ public class Program
 
         app.MapControllers();
 
-        // Allows all origins to access the API.
-        app.UseCors(builder=>builder.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader());
```

Comment:
Is there a reason we swapped this to the localhost version? Feel like hardcoding the localhost into the origins will bite us in the future

Reply:
For security measures, it wouldn't make sense to allow the use of our API from any origins. 
For now the localhost version is only for development purposes and we will definitely create a production policy when the time comes.